examples/javascript/default.js

```diff
@@ -0,0 +1,10 @@
+function add(num1, num2) {
```

Comment:
We try to keep the square() as default example for all languages, is that possible?

Reply:
absolutely. Ill make it a square. I also expect this to go through more changes before it can be merged, but Ill make this change.

Reply:
This is moved to another example file and there it is a square function